### PR TITLE
docs: removes unused dialogflow code samples

### PR DESCRIPTION
**Pull request by Strykrol**
Removes dialogflow samples not surfaced on cloud.google.com. These samples obviated their containing files, which in turn were deleted.

### CODE REVIEW

**Comment by amanda-tarafa** [comment]
Will review when you've been added to the org etc.

**Comment by Strykrol** [comment]
Still working on my CLA with my vendor, thanks for your patience.

**Comment by amanda-tarafa** [comment]
Thanks for the update, no rush on my part.

**Comment by Strykrol** [comment]
@googlebot I signed it!

**Comment by Strykrol** [comment]
I need to rebase and fix commit authoring to trigger the CLA update. Please hold - thank you!

**Comment by amanda-tarafa** [comment]
Friendly ping on this one.

**Comment by Strykrol** [comment]
I will get the tests taken care of tomorrow, sorry for the delay

**Comment by Strykrol** [comment]
Test files removed

File: dialogflow/api/DialogflowSamples/ContextManagement.cs [hunk 1]
```diff
@@ -1,128 +0,0 @@
...
-using System.Collections.Generic;
-using CommandLine;
-using Google.Cloud.Dialogflow.V2;
-
-namespace GoogleCloudSamples
-{
-    // Samples demonstrating how to Create, List, Delete Contexts
-    public class ContextManagement
-    {
-        public static void RegisterCommands(VerbMap<object> verbMap)
```
Comment (amanda-tarafa) [inline dialogflow/api/DialogflowSamples/ContextManagement.cs:26]: I bet this method is being called from somewhere, probably Program.cs, and similar for similar methods on the other deleted samples. Have you made sure that the remaining code still builds after deleting these?

Comment (Strykrol) [reply dialogflow/api/DialogflowSamples/ContextManagement.cs:26]: Thank you for your patience, @amanda-tarafa , and sorry for the delayed response.  Unfortunately I'm running this project across multiple samples repositories (for all languages), and each time requires learning the language/test architecture. I will dig into this and figure out which method references I broke - I'm sure you're right.

Comment (amanda-tarafa) [reply dialogflow/api/DialogflowSamples/ContextManagement.cs:26]: Don't worry, take your time. And let me know if I can help with something.

**Comment by Strykrol** [comment]
Closing in lieu of #1049

**Pull request closed without merging**